lib/GenomeFileUtil/core/GenomeToGenbank.py

```diff
@@ -162,10 +162,8 @@ def __init__(self, genome=None):
         for contig_tuple in contigs_tuples:
             cid = contig_tuple[0]
             feature_ids_by_contig[cid] = {}
-            print("CONTIG ID : " + str(cid))
```

Comment:
I wonder if we should remove these prints (l165 and l168) since that could get chatty for something with lots of contigs.